### DISCUSSION TITLE
previous accumulative cost varies between defualt dashboard and energy dashboard

### DISCUSSION THREAD

**Discussion by potential-difference** [BottlecapDave/HomeAssistant-OctopusEnergy#717] [Q&A]
I've been using this intergration for a while and it's wicked - thank you!!

I just setup a new dashbaord that HA created and manages to see what would happen - to my surprise it created a energy card with the correct cost for yesterdays useage, including the standing charge. screen shots below

I've been unable to get this to happen in the energy dashboard and just resloved to remember to add it in my head when looking at the numbers. However this made me realise that HA is already doing this maths and wonder why it isnt in the energy dashboard already...

has anyone else solved this?

<img width="338" alt="Screenshot 2024-01-21 at 23 48 17" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/38912214/854a3d19-ada4-4beb-8920-dce64d544592">

<img width="797" alt="Screenshot 2024-01-21 at 23 48 49" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/38912214/82784fa7-cf49-4689-b9e9-33f46566caf9">

**Comment by BottlecapDave** [comment]
Hi. The total cost sensors include the standing charge, but if you use the current rate with the energy dashboard then this won't include the standing charge (because it can't due to how HA calculates it). If you follow the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/energy_dashboard/) for how to setup the energy dashboard, then you can have the cost include the standing charge.
